cordova-js-src/splashscreen.js

```diff
@@ -69,18 +69,21 @@ function readPreferencesFromCfg(cfg, manifest) {
         splashImageSrc = schema + ':///' + manifest.getSplashScreenImagePath().replace(/\\/g, '/');
 
         bgColor = cfg.getPreferenceValue('SplashScreenBackgroundColor') || bgColor;
-        bgColor = bgColor.replace('0x', '#').replace('0X', '#');
-        if (bgColor.length > 7) {
-            // Remove aplha
-            bgColor = bgColor.slice(0, 1) + bgColor.slice(3, bgColor.length);
-        }
+        bgColor = bgColor.toLowerCase().replace('0x', '#');
+
+        if (bgColor !== 'transparent') {
+            if (bgColor.length > 7) {
+                // Remove aplha
```

Comment:
maybe fix that typo while you're at it ;)

Reply:
:)